019-ThisOldDataWarehouse/Student/Challenges/Challenge4/README.md

```diff
@@ -11,20 +11,21 @@ Worldwide importers wanted to build out their data warehouse to include clickstr
 Build a streaming pipeline to ingest simulated click stream data into enterprise Delta Lake via Azure Databricks.
 
 ### Data Source: 
-In order to generate the source data stream for this exercise, you will need to execute sample .Net application (Stream Generator).  This code will randomly generate product related data, and write it to an Azure Event Hub.
+In order to generate the source data stream for this exercise, you will need to execute sample .Net application ([Stream Generator](https://github.com/alexkarasek/ClickStreamGenerator)).  This code will randomly generate product related data, and write it to an Azure Event Hub.
+
+Note: You can start the stream of data by executing script below in Azure Cloud Shell:
+
+``` az container create -g [Resource Group Name] --name [container name] --image alexk002/wwiclickstreamgenerator:1 --environment-variables 'hostName'='[EH Host Name]' 'sasKeyName'='RootManageSharedAccessKey' 'sasKeyValue'='[SAS Key]' 'eventHubName'='[Event Hub Name]' ```
```

Comment:
Make it a block, not single line. Looks better.

Reply:
Broke it into multiple lines